omnibus/files/private-chef-cookbooks/private-chef/recipes/config.rb

```diff
@@ -0,0 +1,82 @@
+# Copyright:: Copyright (c) 2015 Chef Software, Inc
```

Comment:
:+1: to having this in its own file , but I'm wondering if we shouldn't follow a convention like `_config.rb` to denote recipes that don't directly change the node?